lib/Myriad/Commands.pm

```diff
@@ -71,7 +71,7 @@ async method service (@args) {
             require_module($module);
             die 'loaded ' . $module . ' but it cannot ->new?' unless $module->can('new');
         } catch ($e) {
-            Future::Exception->throw(sprintf 'Service module %s not found', $module) if $e =~ /Can't locate/;
+            Future::Exception->throw(sprintf 'Service module %s not found', $module) if $e =~ /Can't locate.*($module)/;
```

Comment:
generally when interpolating literals, safer to use [quotemeta](https://perldoc.perl.org/functions/quotemeta):

```suggestion
            Future::Exception->throw(sprintf 'Service module %s not found', $module) if $e =~ /Can't locate.*(\Q$module\E)/;
```